spec/openapi.yaml

```diff
@@ -3950,6 +3997,299 @@ components:
         email: support@mijnwebwinkel.nl
         note: Open from 09:00 till 17:00
         deleted: false
+    Payments:
+      example:
+        id: "8c1ed7ea-c19a-4137-8517-7d3f580b3b97"
+        gateway: "basic"
+        method: "pickup"
+        gatewayReference: "8c1ed7ea-c19a-4137-8517-7d3f580b3b97"
+        price: "4.99"
+        currency: "EUR"
+        storeId: "494c24a3-2243-484a-a81e-b3098d4dc870"
+        reference: "33eea580-2381-44bf-b355-d99a13d4f770"
+        isTest: true
+        createdAt: "2021-06-29T14:41:26+02:00"
+        updatedAt: "2021-06-29T14:41:26+02:00"
+      properties:
+        id:
+          description: Unique identifier
+          type: string
+          format: uuid
+        gateway:
+          $ref: "#/components/schemas/PaymentGatewayList"
+        method:
+          description: Payment method
+          type: string
+        gatewayReference:
+          description: Gateway reference
+          type: string
+        price:
+          description: Price (in decimal notation)
+          type: string
```

Comment:
This is an integer instead, representing minor units of the currency (eg. cents for Euro). Same goes for the order price and order line price.